src/transformers/pipelines/feature_extraction.py

```diff
@@ -61,6 +66,8 @@ def _forward(self, model_inputs):
 
     def postprocess(self, model_outputs):
```

Comment:
The default argument needs to be here. 

Check out https://huggingface.co/docs/transformers/v4.22.2/en/add_new_pipeline#how-to-create-a-custom-pipeline for more info on the code structure.